apps/files/src/components/Collaborators/SharedFilesList.vue

```diff
@@ -45,6 +45,14 @@
       </div>
       <div class="uk-text-meta uk-text-nowrap uk-width-small" v-text="formDateFromNow(item.shareTime)" />
     </template>
+    <template #noContentMessage>
+      <no-content-message icon="share">
+        <template #message>
+          <span v-if="$_isSharedWithMe"><translate>You are currently not collaborating on other people's resources.</translate></span>
```

Comment:
`translate` tags default to span. Could we then either use `v-translate` directly on the span or dropping the span and moving the if statement to the translate tag? I'd prefer the former.